module-utils/log/Logger.cpp

```diff
@@ -20,6 +20,7 @@ namespace Log
                                                             {"ServiceAntenna", logger_level::LOGERROR},
                                                             {"ServiceAudio", logger_level::LOGINFO},
                                                             {"ServiceBluetooth", logger_level::LOGINFO},
+                                                            {"ServiceBluetooth_w1", logger_level::LOGINFO},
```

Comment:
needed?

Reply:
covinience to not log debug information added while using the Bluetooth.
Tbh it would be great to have window to set these in i.e. settings and be able to change it in runtime

Reply:
We have a Jira for that, @pholat :)